c2corg_ui/templates/helpers/view.html

```diff
@@ -0,0 +1,117 @@
+%>
+
+<%def name="show_attr(obj, key, parse=True)">\
+  <%
+      attr = get_attr(obj, key, md=parse, bb=parse)
+      attr = attr if attr else ''
+  %>
+  ${attr | n}\
+</%def>
+
+<%def name="show_short_attr(obj, attr, type='raw', suffix='')">\
+  % if attr in obj and obj[attr]:
+      % if type == 'i18n':
+        <span class="detail-value">{{mainCtrl.translate('${obj[attr]}')}}</span>
+      % elif type == 'list':
+        <span class="detail-text" ng-repeat="val in ['${"','".join(obj[attr]) |n}']">{{mainCtrl.translate(val)}}</span>
+      % elif type =='text':
+        <span class="detail-text">${show_attr(obj, attr)}</span>
+      % else:
+        <span class="detail-text">${obj[attr]}${suffix}</span>
+      % endif
+  % endif
+</%def>
+
+<%def name="show_archive_data(module, document, locale, version)">\
+  <ul class="archive-data">
+    <li translate>You are viewing an archived version of this document.</li>
+    <li>{{'${1000 * version['written_at']}' | date:'medium'}} - <a href="#TODO">${version['username']}</a> - ${show_version_comment(version)}</li>
+    <li><a href="${request.route_url(module + '_view', id=document['document_id'], lang=locale['lang'])}" translate>See the latest version</a></li>
+  </ul>
+</%def>
+
+<%def name="show_other_langs_links(module, document, other_langs)">\
+  % if other_langs:
+    <div id="other-langs">
+      <div class="modal-header"><h3 class="text-center" translate>View in other lang</h3></div>  
+      <ul>
+      % for l in other_langs:
+        <li>
+          <span href="${request.route_url(module + '_view', id=document['document_id'], lang=l)}">{{mainCtrl.translate('${l}')}}</span>
+           <span class="glyphicon glyphicon-chevron-right"></span>
+        </li>
+      % endfor
+      </ul>
+    </div>
+  % endif
+</%def>
+
+<%def name="show_missing_langs_links(module, document, missing_langs)">\
+% if missing_langs:
+  <div  id="missing-langs">
+    <div class="modal-header"><h3 class="text-center" translate>Translate into an other lang</h3></div>  
+    <ul>
+      % for lang in missing_langs:
+      <li>
+      <app-edit-button app-edit-button-url="${request.route_url(module + '_edit', id=document['document_id'], lang=lang)}">
+        {{mainCtrl.translate('${lang}')}}
+        <span class="glyphicon glyphicon-chevron-right"></span>
+      </app-edit-button>
+      </li>
+      % endfor
+    </ul>
+</div>
+% endif
+</%def>
+
+<%def name="show_route_title(locale, is_tab_title=False)">\
+  <%
+      title = ''
+      if 'title_prefix' in locale and locale['title_prefix']:
+          title = locale['title_prefix'] + ' : '
+      title += locale['title']
+  %>\
+${title}${' - Camptocamp.org' if is_tab_title else ''}\
+</%def>
+
+<%def name="show_version_comment(version)">\
+  ${'{{mainCtrl.translate(\'' + get_attr(version, 'comment', md=False) + '\')}}' if version['comment'] else '' | n}
+</%def>
+
+<%def name="create_attribute_element_value(type, attr)">\
```

Comment:
Those 3 helpers seem no longer used (replaced by ``show_short_attr`` defined a little above in the file).